crates/hyperswitch_connectors/Cargo.toml

```diff
@@ -30,9 +30,11 @@ ring = "0.17.8"
 serde = { version = "1.0.197", features = ["derive"] }
 serde_json = "1.0.115"
 serde_urlencoded = "0.7.1"
+serde_with = "3.7.0"
 strum = { version = "0.26", features = ["derive"] }
 time = "0.3.35"
 url = "2.5.0"
+urlencoding = "2.1.3"
```

Comment:
Can we confirm if we can use the [`url_encode()`](https://github.com/juspay/hyperswitch/blob/8372389671c4aefeb625365d198390df5d8f35a5/crates/common_utils/src/ext_traits.rs#L122) utility method instead of the `urlencoding` crate?

Reply:
This function [url_encode()](https://github.com/juspay/hyperswitch/blob/8372389671c4aefeb625365d198390df5d8f35a5/crates/common_utils/src/ext_traits.rs#L122) only serializes supports only maps and structs, hence we cannot use this.